contracts/netmap/contract.go

```diff
@@ -516,6 +611,9 @@ func UpdateSnapshotCount(count int) {
 		key := snapshotKeyPrefix + string([]byte{byte(k)})
 		storage.Delete(ctx, key)
 	}
+	for k := Epoch() - curr + 1; k < Epoch()-count; k++ {
```

Comment:
do u intentionally recall `Epoch()` each time? guess u wanna cover a potential parallel tick. If so, i'd write a comment

Reply:
No, the epoch can't change here, I'll optimize this a bit.